stock_picking_progress/models/stock_picking.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import api, fields, models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    progress = fields.Float(compute="_compute_progress", store=True)
+
+    @api.depends("move_line_ids", "move_line_ids.progress")
+    def _compute_progress(self):
+        for record in self:
+            lines_progress = record.move_line_ids.mapped("progress")
```

Comment:
the value on the move lines is stored: can't you do one select only w/ sum + count?